xscen/data/file_schema.yml

```diff
@@ -75,35 +92,58 @@ raw-hydro-reconstruction:
     - hydrology_source
     - institution
     - [source, version]
-    - member
+    - (member)
     - frequency
     - variable
   filename: [ variable, frequency, domain, hydrology_project, hydrology_institution, hydrology_source, institution, source, version, member, DATES ]
-raw-hydro-sims:
+raw-hydro-sims-raw:
+  with:
+    - facet: type
+      value: simulation-hydro
+    - facet: processing_level
+      value: [ raw ]
+  folders:
+    - type
+    - hydrology_project
+    - hydrology_institution
+    - hydrology_source
```

Comment:
`hydrology_member` is missing from this list. Also, after reflection, I think that similar to how `bias_adjust_institution` is in the catalog but not the structure, we could remove `hydrology_institution`. So:

```
    - hydrology_project
    - hydrology_source
    - (hydrology_member)
```